wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -1,8 +1,21 @@
 # Nomination Assessment Team
 
-The **Nomination Assessment Team** (***NAT***) are the moderators of the [Beatmap Nominators](/wiki/People/Beatmap_Nominators). They are responsible for reviewing BN applications, creating tests for the applicants, evaluating current members and overseeing vetoes in the ranking process. All NAT members may act as BNs in the modding and ranking process, but they do not have strict activity requirements like BNs.
+The **Nomination Assessment Team** (**NAT**) are the moderators of the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (BN) and ensure the beatmapping side of osu! is functional.
+
+Members of the NAT are distinguished by their red username and user title. Like the [Global Moderation Team](/wiki/People/Global_Moderation_Team) (GMT), they also have site-wide moderation permissions and a dark red username in the game's chat channels.
+
+## Responsibilities
+
+The NAT are responsible for a variety of mapping-related tasks that can be separated into four subcategories:
+ - **Evaluation:** Reviewing BN applications and evaluating proficiency of current Beatmap Nominators. Most members of the NAT are solely responsible for contributing to this subcategory.
```

Comment:
"core" is an odd term to use if you don't explain it before this

Reply:
"Evaluating the Beatmap Nominators and its applications" -- doesn't sound great, because BNs refers to the people in the group, but then "it" refers to the group

Reply:
I haven't looked over the rest of the stuff, but considering these are all **cores** we're explaining here, it seems very odd not to explain (or at least link to a page that explains) what a core is. Hint: They are not responsibilities, not sure why the subtitle makes them seem like that; the people listed under each core solely _contribute_ to it, they are not responsible for all encompassing tasks. Seems misleading, should probably sort that out @pishifat .

As for the other thing, the group is called "the Beatmap Nominators" so "it" is referring to the group. In the past we used the term "the Beatmap Nomination Group" (which was arguably less confusing in situations like these) but it was later changed to what it is now.

Reply:
sentence uses it twice in different ways. it's fine to refer to it as either, but not both at the same time if you can avoid it

Reply:
oh true, missed that part, reworded

Reply:
applied but used "subcategory" instead of "core" since core doesnt have the same inherent meaning

Reply:
need to fix Moderation in that case